Docker.DotNet.sln

```diff
@@ -1,39 +1,93 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25123.0
+# Visual Studio 15
+VisualStudioVersion = 15.0.26020.0
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Docker.DotNet", "Docker.DotNet\Docker.DotNet.xproj", "{A9C3E587-DDEC-494B-96E2-BFB4FC34EC74}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Docker.DotNet", "Docker.DotNet\Docker.DotNet.csproj", "{A9C3E587-DDEC-494B-96E2-BFB4FC34EC74}"
 EndProject
-Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Docker.DotNet.BasicAuth", "Docker.DotNet.BasicAuth\Docker.DotNet.BasicAuth.xproj", "{EC789466-95C7-4FAB-9575-18B2FF55ABD3}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Docker.DotNet.BasicAuth", "Docker.DotNet.BasicAuth\Docker.DotNet.BasicAuth.csproj", "{EC789466-95C7-4FAB-9575-18B2FF55ABD3}"
 EndProject
-Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Docker.DotNet.X509", "Docker.DotNet.X509\Docker.DotNet.X509.xproj", "{3EB8C108-EAD7-4A58-AACF-33D9149F4708}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Docker.DotNet.X509", "Docker.DotNet.X509\Docker.DotNet.X509.csproj", "{3EB8C108-EAD7-4A58-AACF-33D9149F4708}"
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{9ADC524F-FC79-4D05-9536-37BB4D426C0F}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Docker.DotNet.Tests", "Docker.DotNet.Tests\Docker.DotNet.Tests.csproj", "{3225A078-3F6C-4293-B580-1EA3203EA3E8}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "NuGet", "NuGet", "{5B3E2DDE-7EF4-4E55-A0BF-05EF43048336}"

```

Comment:
This is new to me. Why add the nuspec's to the solution? Is that a new csproj requirement?